### ISSUE TITLE
Increase the max height of pistons

### ISSUE THREAD

**Issue by Konclan** [TeamSpen210/BEE2-items#492]
Would it be possible to increase the maximum height that pistons can go. Their max height atm is kinda lame because it is very small compared to some chambers people make.

**Comment by TheReviewers** [comment]
@TeamSpen210 I dont think that is possible...

**Comment by Konclan** [comment]
@TheReviewers Yeah I have a feeling it may not be as well

**Comment by TheReviewers** [comment]
@Konclan What happened to the other comments?

**Comment by Konclan** [comment]
hmm?

**Comment by TeamSpen210** [comment]
I could make a second piston platform item which is double the length or something similar. I don't think I will though. If the piston would be any longer, it'd be a bit unrealistic - an unsupported piston that long in real life would probably start to bend or droop if placed sideways or on the floor.

**Comment by TheReviewers** [comment]
@TeamSpen210 But teamspen, in real life you couldnt spawn black holes, in real life you cant jump through a blue funnel made of asbestos etc...

**Comment by Konclan** [comment]
Blue funnel of asbestos? Excursion Funnels are not made of asbestos, the only mention of asbestos was in the old aperture science spheres.

**Comment by TheReviewers** [comment]
@Konclan You obviously didnt watch the excursion funnel trailer made by valve

Find it on youtube

**Comment by Konclan** [comment]
Well I learned something new

**Comment by Luke18033** [comment]
@TeamSpen210 Portal 1 has huge pistons in the final sections of Test Chamber 18 though...

**Comment by TeamSpen210** [comment]
True, those are really thick and vertical though. For Clean I might make the piston base surrounded by a black tile pillar.
